### PR TITLE
Display warning text when address / boundary geojson has been manually updated

### PR DESCRIPTION
**Pull request by benbaumann95**
### Description of change

- Display warning text when address / boundary geojson has been manually updated

### Story Link

https://trello.com/c/OpJtIGV9/765-include-a-prompt-when-address-or-polygons-are-updated-to-check-constraints-and-proposal-details

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -237,6 +237,8 @@
     t.datetime "closed_at"
     t.jsonb "planx_data"
     t.datetime "determination_date"
+    t.integer "applicant_role"
```
Comment (EGiataganas) [inline db/schema.rb:240]: This shouldn't be in this schema 😄

File: app/models/planning_application.rb [hunk 2]
```diff
@@ -488,6 +497,14 @@ def audit_updated!
     end
   end
 
+  def address_or_boundary_geojson_updated?
+    return if updated_address_or_boundary_geojson
+
+    if saved_changes.keys.intersection(ADDRESS_AND_BOUNDARY_GEOJSON_FIELDS).any?
+      update!(updated_address_or_boundary_geojson: true)
+    end
```
Comment (EGiataganas) [inline app/models/planning_application.rb:505]: How about add an audit for this?

Comment (benbaumann95) [reply app/models/planning_application.rb:505]: The updates to any of those fields are already captured in the audit log so hopefully that should cover this

File: db/schema.rb (hunk 1 above)
Comment (benbaumann95) [reply db/schema.rb:240]: woops !

**Review by EGiataganas — APPROVED** [review]

**Pull request merged**